docs/tutorials/sources/file.rst

```diff
@@ -0,0 +1,82 @@
+.. _new_file_type_source_tutorial:
+
+Source for New file types
+=========================
+
+This tutorial will help you with implementing your own ``Source`` for new file
+types. You may want to do this if you have some specific file formats from which
+you want to load and save data from. Here we will go through implementing a source
+for ``.ini`` file formats.
+
+About INI files
+---------------
+
+An INI file is a configuration file used by Operating Systems and programs
+to initialize program settings. It contains sections for settings and preferences
+(delimited by a string in square brackets) with each section containing one
+or more name and value parameters.
+
+Import modules
+--------------
+
+.. literalinclude:: ../../../dffml/source/ini.py
+    :lines: 1-8
```

Comment:
Recently we made it so that we can import everything `from dffml`. However, we can't do that from within DFFML itself.

Let's have this be a `.. code-block:: python` instead

```suggestion
.. code-block:: python

    from configparser import ConfigParser

    from dffml import config, entrypoint, Record, FileSource, MemorySource, parser_helper
```

Reply:
done